### PR TITLE
fix: update of the population script

### PR DESCRIPTION
**Pull request by Jatewo**
## Description

Update of the population script to add pollingStations. It also introduces an eslint check for jsdoc comments.

## Changes made
- Added polling stations to db populate script
- Removed duplicate names and duplicate objects from mockdata
- Added eslint check for jsdoc comments

### CODE REVIEW

File: config/cors.js
Comment (MortenMunk) [inline config/cors.js:1]: Perhaps use PORT from .env but it doesnt really matter imo

**Review by MortenMunk — APPROVED** [review]
mostly documentation and linting so i approve

**Review by Mafusn — APPROVED** [review]
LGTM

**Review by TheMOHD3886 — APPROVED** [review]
lgtm 👍

**Pull request merged**